src/ui/src/builder/panels/BuilderBlueprintLibraryPanel.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+	<WdsModal
+		v-if="isOpen && isBlueprintLibraryEnabled"
```

Comment:
It is not needed since you're already checking that in the parent
```suggestion
		v-if="isOpen"
```